src/components/Onfido/BaseOnfidoWeb.js

```diff
@@ -83,13 +85,28 @@ class Onfido extends React.Component {
             ],
             smsNumberCountryCode: CONST.ONFIDO.SMS_NUMBER_COUNTRY_CODE.US,
             showCountrySelection: false,
-            onComplete: this.props.onSuccess,
+            onComplete: (data) => {
+                if (_.isEmpty(data)) {
+                    Log.warn('Onfido completed with no data');
+                }
+                this.props.onSuccess(data);
+            },
             onError: (error) => {
                 const errorMessage = lodashGet(error, 'message', CONST.ERROR.UNKNOWN_ERROR);
+                Log.hmmm('Onfido error', {errorType: error.type, errorMessage});
```

Comment:
NAB, not sure if this is possible but if `error` is `undefined` this could throw. Then again, maybe we trust Onfido to always pass something to this callback... 😄